src/routes/plans/[id]/+page.svelte

```diff
@@ -104,7 +107,26 @@
     $view = { ...data.initialView };
   }
 
-  $: $activitiesMap = createActivitiesMap($plan, $activityDirectives, $simulationSpans);
+  $: if ($plan) {
+    effects
+      .getResourcesExternal($plan.id, $plan.start_time, $plan.duration)
+      .then(newResources => ($externalResources = newResources));
+  }
+
+  $: if ($plan && $simulationDataset !== undefined) {
+    if ($simulationDataset !== null) {
+      const datasetId = $simulationDataset.id;
```

Comment:
I _think_ we want `dataset_id` here:
```suggestion
      const datasetId = $simulationDataset.dataset_id;
```

Reply:
Wow 😵‍💫
Great catch - updated